demos/qadi_experiments/qadi_prompt.py

```
@@ -0,0 +1,97 @@
+env_file = Path(__file__).parent / '.env'
+if env_file.exists():
+    with open(env_file) as f:
+        for line in f:
+            if line.strip() and not line.startswith('#') and '=' in line:
+                key, value = line.strip().split('=', 1)
+                os.environ[key] = value.strip('"').strip("'")
+
+# Add src to path
+sys.path.insert(0, str(Path(__file__).parent / 'src'))
+
+from mad_spark_alt.core.enhanced_orchestrator import EnhancedQADIOrchestrator
```

Ruff (0.11.9)

Module level import not at top of file

(E402)